recipe/meta.yaml

```diff
@@ -30,6 +30,7 @@ test: {}
 about:
   home: https://github.com/conda-forge/blas-feedstock
   license: BSD 3-clause
+  license_file: {{ RECIPE_DIR if RECIPE_DIR is defined else '' }}/LICENSE.txt
```

Comment:
Checking if a variable is defined like this seems to work really well for the linter apparently. Just to muse on this a little bit, I can see a lot of value in having a jinja module loaded in advance by `conda-smithy` that mocks out a bunch of these variables using something like this.